File: Lab2/pzpi-22-3-protsenko-artem-lab2/PetHouse.Application/Contracts/Meal/MealDto.cs
using PetHouse.Core.Enums.Meal;

namespace PetHouse.Application.Contracts.Meal;

public class MealDto
{
   public Guid MealId { get; set; }
   public Guid PetId { get; set; }
   public double  PortionSize { get; set; }
   public bool IsDaily { get; set; }
   public double CalorificValue { get; set; }
   public double  CaloriesPerMeal { get; set; }
   public DateTime StartTime { get; set; }
   public MealStatus MealStatus { get; set; }
}

File: Lab4/pzpi-22-3-protsenko-artem-lab4/app/src/main/java/com/pethouse/pethousemobile/models/Meal/MealDto.kt
package com.pethouse.pethousemobile.models.Meal

data class MealDto(
    val mealId: String,
    val petId: String,
    val portionSize: Int,
    val startTime: String,
    val caloriesPerMeal: Double,
    val caloriesConsumed: Int,
    val adaptiveAdjustment: Boolean,
    val foodType: String,
    val calorificValue: Int,
    val mealStatus: String,
    val isDaily: Boolean
)
